### PR TITLE
New category Election and two additions to the Editor category

### LINKED ISSUE #23: Add more icons
- [x] **/editor/** exact match filter icon
- [x] **/elections/** new category

### PR DESCRIPTION
**Pull request by sdimchevski**
- New **Election** category with 14 new icons added
- New **exact match** and **match** icons added to the Editor category
closes #23

### CODE REVIEW

File: src/svgs/exact-match-1.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M5.683,9a7,7,0,1,1,0,6H8.025a5,5,0,1,0,0-6ZM3.79,7.047a9.6,9.6,0,1,1,0,9.906L2.055,18.688a12,12,0,1,0,0-13.376ZM0,14.914l1.432,1.414L4.8,13h6.161v2L14,12,10.963,9v2H4.8L1.432,7.672,0,9.086,2.95,12Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/exact-match-1.svg]: Remove this one as it is a duplicate of the one below exact-match.svg

File: src/svgs/results-by-candidates.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M11,12.315V2.062a9.985,9.985,0,1,0,5.865,18.631ZM7.3,10a1.351,1.351,0,1,1-1.35,1.35A1.35,1.35,0,0,1,7.3,10ZM10,15.4H4.6v-.676c0-.9,1.8-1.35,2.7-1.35s2.7.452,2.7,1.35Zm3-13.34v9.623l5.5,7.863A9.956,9.956,0,0,0,13,2.062ZM17,8a1,1,0,1,1-1,1A1,1,0,0,1,17,8Zm2,4H15v-.5c0-.665,1.332-1,2-1s2,.335,2,1Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/results-by-candidates.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-ca.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M8.494,3.285,7.143,8.632l6.338,8.946L14,19.121l-1.247,1.748.312.823L12.234,22a7.152,7.152,0,0,0-3.325-.411,13.7,13.7,0,0,0-.26-2s-.727-.823-1.194-.772c-.143.016-.052-.565-.052-.565l-.624-.1-.935-1.028L4.494,16.7l.415-1.234L3.247,13l.623-1.132s-.727.309-.727-.719,0-1.543-.52-1.748l.52-.515s-.931,0-1.039-.822c-.156-1.183.519-1.543.519-2.468C2.623,5.085,2,5.393,2,4.468,2,3.851,2.727,4.057,3.039,2ZM10.542,8.2a3.212,3.212,0,0,0,.7,2.2,2.445,2.445,0,0,0,1.932.8,2.3,2.3,0,0,0,1.629-.59A2.6,2.6,0,0,0,15.6,9.07H14.4a1.817,1.817,0,0,1-.3.668,1.128,1.128,0,0,1-.944.414,1.225,1.225,0,0,1-1-.483,2.262,2.262,0,0,1-.392-1.451,2.581,2.581,0,0,1,.371-1.5,1.174,1.174,0,0,1,1.014-.529,1.186,1.186,0,0,1,.953.375,1.647,1.647,0,0,1,.295.625h1.2a2.084,2.084,0,0,0-.394-1.074,2.384,2.384,0,0,0-2.1-.953,2.367,2.367,0,0,0-1.77.711A3.182,3.182,0,0,0,10.542,8.2Zm5.619,2.832h1.258l.391-1.184h2.12l.38,1.184h1.3L19.578,5.273H18.216Zm2.723-4.442.728,2.266H18.137Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-ca.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-fl.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M7.927,2.455l.616.909,5.389-.606s.616.606,1.078.454S15.318,2,15.318,2h.615c2,5.152,3.7,5.606,3.7,5.606V8.97a8.284,8.284,0,0,1,2.31,4.848,2.06,2.06,0,0,1-.77,1.364l.462,1.06L20.09,17l-.923-1.818-1.386-.909s-2.31-2.576-2.771-3.182c.615-1.212,0-1.212,0-1.212l-.462.454-.308-.606c.924-2.272-.308-3.03-.308-3.03l-.77-.152A3.906,3.906,0,0,0,9.852,4.5l-2,1.515A6.484,6.484,0,0,0,5.541,4.2a5.849,5.849,0,0,0-3.233.3L2,3.439ZM2.279,21.516h1.2V19.1h2.55v-1H3.475V16.777h2.91V15.766H2.279Zm4.887,0h4.078V20.48H8.369V15.758h-1.2Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-fl.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-ny.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M2.115,2.758H3.376L5.662,6.772V2.758H6.783V8.516H5.58L3.236,4.43V8.516H2.115Zm9.082,0h1.364l-1.926,3.6v2.16h-1.2V6.356l-1.993-3.6H8.857l1.2,2.508ZM14.366,7.9l-2.443,3.6-.2,2L6.047,15.207a.7.7,0,0,0-.489.869c.082.266.863.448.665,1.224a4.16,4.16,0,0,1-2.035,2.4l1.017.8,11.4-2,1.222,1,2.646,1.2.916,1.3L22,20.6l-.407-6.2L19.15,7Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-ny.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-party-democratic.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M19,8H11.189L7.766,4.753S8.024,2,7.08,2c-.686,0-.943,1.285-1.2,2.111C5.194,3.468,3.842,2.358,3.327,3S4.594,5.579,4.594,5.579c-.972.382-.54,1.064-1.505,1.577A2.6,2.6,0,0,0,2.5,10.794c.907.814,3.541-.389,3.773-.794C6.876,11.193,8,14,8,14v8h2V20l1.249.975,1.39-1.414-1.911-1.944V17h5.8L15.4,20.571l1.87.615L18,19.38V22h2.034V12.371L21.017,14s.825-.849.983-1ZM9.638,11.516l-.747-.552-.748.552.286-.894-.748-.553h.924l.286-.894.285.894H10.1l-.748.553Zm3.145,0-.747-.552-.748.552.286-.894-.748-.553h.924l.286-.894.285.894h.925l-.748.553Zm3.145,0-.747-.552-.748.552.285-.894-.747-.553H14.9l.286-.894.285.894h.924l-.747.553Zm3.145,0-.747-.552-.748.552.285-.894-.747-.553h.924l.286-.894.285.894h.924l-.747.553Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-party-democratic.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-party-green.svg [hunk 7]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M21.361,10.458,13.542,2.639a2.18,2.18,0,0,0-3.084,0L2.639,10.458a2.18,2.18,0,0,0,0,3.084l7.819,7.819a2.18,2.18,0,0,0,3.084,0l7.819-7.819A2.18,2.18,0,0,0,21.361,10.458Zm-4.838,4.127c-.295,1.236-.232,3.127-2.037,2.569-.023-.288-.044-.569-.07-.894-8.755,4.272-10.878-10-2.144-10.017,1.532,0,4.814,1.231,3.6,3.239-3.331-2.939-8.5-.694-6.974,4,1.131,2.245,4.766,2.221,6.007.125h-2.7a1.285,1.285,0,0,1,.1-1.355c.878-.387,4.318-.483,4.7-.118C16.844,12.939,16.689,13.763,16.523,14.585Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-party-green.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-party-libertarian.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...5,6,16.623,4.6,16.864,4,14,4,13.376,7.48,13.355,7.517a1.7,1.7,0,0,1-1.017-.84,3.026,3.026,0,0,1,.073-2.609,3.481,3.481,0,0,0-1.622,1.165,3.5,3.5,0,0,0-.826,2.645A3.96,3.96,0,0,1,8.169,5.233c-1.222,1-1.278,2.269-.929,3.684A2.9,2.9,0,0,1,4.512,7.149C4,9,5.116,10.524,5.934,11.758c-.934.242-2-.743-2.962-1.2a2.515,2.515,0,0,0,1.349,2.19,1.331,1.331,0,0,0,.592.2q7.627.017,15.255.007c.106,0,.212-.016.331-.026C19.841,11.548,20.217,10.512,21.9,9.034ZM10.108,12.242l-.93-.678-.905.687c.116-.445.209-.8.3-1.162L7.7,10.377l1.11-.059c.113-.347.223-.682.363-1.111.141.407.257.739.389,1.122h1l.029.094-.806.667C9.889,11.46,9.987,11.809,10.108,12.242Zm3.71.008-.923-.686-.919.687c.117-.435.208-.774.309-1.152l-.777-.643.01-.1h.991c.122-.378.229-.712.374-1.161.146.434.258.769.383,1.14h.991l.039.1-.8.658C13.6,11.448,13.693,11.8,13.818,12.25Zm3.706,0-.922-.687-.922.691c.12-.445.212-.784.313-1.157l-.863-.711,1.093-.071c.112-.339.227-.684.369-1.113.139.41.252.746.381,1.126h1.009l.032.091-.809.668Zm3.7,1.75L22,15.192c.182,2.186-2.589,2.377-4.154,2-.375,1.116.54,1.883.934,2.809H16.327c-.326-2.361-6.415-2.772-3.986,0H9.505C8.493,17.444,6,20,3.77,16.858a3.23,3.23,0,0,0,2.692-.665C4.647,16.122,2,15,2,14Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-party-libertarian.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-party-republican.svg [hunk 9]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M23,17v1a3,3,0,0,1-6,0V15.5a.5.5,0,0,0-.5-.5h-1a.5.5,0,0,0-.5.5V21H11V17H5v4H1V12H19v6a1,1,0,0,0,2,0V17ZM19,7.5V10H1V7.5C1,5.015,5.029,3,10,3S19,5.015,19,7.5ZM6.332,7.727,5.363,7c.124-.413.242-.8.36-1.19l-.039-.024-.969.732-.989-.724-.03.026c.124.394.247.787.374,1.19l-1.017.787H4.316c.135.4.266.794.4,1.184h.041c.122-.393.244-.787.373-1.2H6.317Zm5.031,0-.975-.733c.126-.411.245-.8.365-1.188l-.04-.026c-.327.246-.653.493-.972.732L8.7,5.743,9.1,7l-.958.746.018.046H9.345L9.74,8.977h.038l.373-1.207h1.2Zm5.117.051c-.324-.242-.647-.485-.976-.732.119-.408.235-.8.351-1.2l-.035-.022-.969.73-.988-.726-.028.028c.121.389.241.778.369,1.19L13.24,7.8l.016.037h1.2c.131.4.26.8.389,1.2l.041-.005c.122-.395.243-.791.37-1.2h1.209Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-party-republican.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

File: src/svgs/usa-tx.svg [hunk 10]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.a{fill:none;}</style></defs><path class="a" d="M0,0H24V24H0Z"/><path d="M10.987,2l3.237.235V4.742l1.618.744,2.329.744,1.5-.314,1.461.431.079,1.88L22,9.48l-.158,1.959-1.105.626L19.4,13.084l-2.132.548s-.4,2.741.158,3.368l-2.447-.627L14.5,14.728l-1.974-3.211-1.421-.235-.947,1.253S8.5,11.752,8.342,10.89s-.079-1.41-.868-1.41c-.553-.548-1.263-1.331-1.263-1.331H10.4ZM2.469,15.758v1.019H4.2v4.739H5.41V16.777H7.133V15.758Zm7.442,3.794,1.1,1.964h1.461L10.6,18.535l1.761-2.777h-1.39L9.911,17.633,8.885,15.758H7.447l1.762,2.824L7.357,21.516h1.4Z"/></svg>
```
Comment (StefanIvanov) [inline src/svgs/usa-tx.svg]: I think we should remove this part: <defs><style>.a{fill:none;}</style></defs>

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — CHANGES_REQUESTED** [review]
Now you only need to remove src/svgs/exact-match-1.svg

**Review by StefanIvanov — APPROVED** [review]

**Review by simeonoff — APPROVED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Review by simeonoff — APPROVED** [review]

**Pull request merged**